src/app.js

```diff
@@ -25,8 +39,58 @@ const quoteData = [
   },
 ];
 
+const quoteList = new QuoteList();
+const tradeList = new TradeList();
+const openOrderList = new OpenOrderList();
+
+let quoteTemplate
+let tradeTemplate
+let orderTemplate
+
+quoteData.forEach(function(datum) {
+  quoteList.add(new Quote(datum))
```

Comment:
It is unclear to me why we're creating this separate instance of `QuoteList`. In the `$(document).ready` callback there is already an instance created from `quoteData` (it's the `quotes` variable).

Because we're creating this separate `quoteList` variable and passing it to `new QuoteListView`, and the `Simulator` model is receiving the other `quotes` variable, there is a disconnect between which quotes are having their price change and which quotes are being displayed on the page.

You might have noticed that none of the prices for your quotes are changing automatically, and this is why.